scripts/onboard.js

```diff
@@ -0,0 +1,260 @@
+var grid, refreshHandle = 99;
```

Comment:
Because this file gets generated by the TS compiler, there is no need to commit it